geocoordinates/geocoordinates.php

```diff
@@ -26,34 +26,41 @@ function geocoordinates_uninstall()
 
 function geocoordinates_resolve_item(&$item)
 {
-	if((!$item["coord"]) || ($item["location"]))
+	if (!isset($item['coord']) || isset($item['location'])) {
```

Comment:
I have not touched this logic part, but isn't a `!` missing here?

Reply:
Ah, oh. Big cinema: type-hint `array` is missing!

Reply:
In any case you shouldn't use ```isset``` but ```empty```. ```isset``` will be true for an empty string - which would create side effects.

And no, this query is okay. We don't need to do a resolve for a location when a location is already there.

Reply:
Okay, I rewrite this part.

Reply:
So `if (empty($item['coord']) || !empty($item['location'])) {` is okay?

Reply:
Yes, it is.

Reply:
Done.